### ISSUE TITLE
Tap targets are not sized appropriately

### ISSUE THREAD

**Issue by spaceninja** [MLTSHP/mltshp-patterns#256]
Interactive elements like buttons and links should be large enough (48x48px), and have enough space around them, to be easy enough to tap without overlapping onto other elements. [Learn more](https://web.dev/tap-targets/).

Failing Elements
--
- Every "pro badge" plus icon.
- Several links in the footer

These issues can be [addressed with CSS](https://ishadeed.com/article/clickable-area/).

You can test the results using Lighthouse.
